rosette/default.nix

```diff
@@ -16,6 +16,7 @@ in pkgs.stdenv.mkDerivation {
     [ cmake
       doxygen
       pkgconfig
+      protobuf
```

Comment:
I think you should put this in `buildInputs` instead of `nativeBuildInputs`.

Reply:
Relevant bits:

**nativeBuildInputs**
> programs/libraries used at build time that, if they are a compiler or similar tool, produce code to run at run time—i.e. tools used to build the new derivation.

I'm following this definition and nixpkgs convention when it comes to putting `cmake`, `doxygen`, and `pkgconfig` here.

**buildInputs**
> These often are programs/libraries used by the new derivation at run-time, but that isn't always the case. For example, the machine code in a statically linked library is only used at run time, but the derivation containing the library is only needed at build time. Even in the dynamic case, the library may also be needed at build time to appease the linker.


I suggested putting `protobuf` in `buildInputs` because I'm assuming that, in addition to the `protoc` compiler which generates code, there are probably protobuf runtime support libraries that need to be linked for use at runtime.  I've done similar things with the desired outcome for the C++ version of CapnProto.